examples/dask_ml_simple.py

```diff
@@ -0,0 +1,76 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset using Dask-ML.
+
+In this example, we optimize a logistic regression classifier configuration for the Iris dataset.
+The classifier is from dask-ml while the dataset is from sklearn.
+We optimize the choice of solver (admm, gradient descent, or newton),
+the regularization (penalty) when relevant and its strength (C).
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python dask_ml_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize dask_ml_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import dask.array as da
+from dask_ml.linear_model import LogisticRegression
+from dask_ml.model_selection import train_test_split
+from distributed import Client
+from distributed import LocalCluster
+from sklearn.datasets import load_iris
+
+
+# FYI: Objective functions can take additional arguments
+# (https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args).
+def objective(trial):
+    iris = load_iris()
+    X, y = iris.data, iris.target
+    X, y = da.from_array(X, chunks=len(X) // 5), da.from_array(y, chunks=len(y) // 5)
+
+    solver = trial.suggest_categorical('solver', ['admm', 'gradient_descent', 'proximal_grad'])
+    C = trial.suggest_uniform('C', 0.0, 1.0)
+
+    if solver == 'admm' or solver == 'proximal_grad':
+        penalty = trial.suggest_categorical('penalty', ['l1', 'l2', 'elastic_net'])
```

Comment:
[PERSONAL NOTE] Inside dask-ml, [`Regularizer.get(penalty)`](https://github.com/dask/dask-glm/blob/master/dask_glm/regularizers.py#L125) is called to create an instance of `Regularizer` from the penalty name.

Reply:
Do you want me to add that as a comment? I would expect people who want to use Dask-ML to find that in the docs instead

Reply:
No, this is just a note for me.

Because [the document of `LogisticRegression`](https://ml.dask.org/modules/generated/dask_ml.linear_model.LogisticRegression.html) doesn't mention to "elastic_net" (it says "For string values, only ‘l1’ or ‘l2’ are valid."), I read the source code to find out the usage of `penalty` string.

Reply:
I think the docs are not in sync with the code.

[Here](https://github.com/dask/dask-ml/blob/master/dask_ml/linear_model/glm.py#L137,L169) we can see where the penalty attribute is used in LogisticRegression. There is no check for elastic_net and no exception is raised when it is passed.

[That method](https://github.com/dask/dask-ml/blob/master/dask_ml/linear_model/glm.py#L137,L169) is called in the [fit method](https://github.com/dask/dask-ml/blob/master/dask_ml/linear_model/glm.py#L171,L199) and its result is passed to a solver from [this dictionary of solvers](https://github.com/dask/dask-glm/blob/master/dask_glm/algorithms.py#L434).

Since Elastic Net is just the weighted average of L1 and L2 it should work of both L1 and L2 work.